themes/default/content/docs/intro/concepts/plans.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Plans"
+meta_desc: Learn about Pulumi update plans and how they can be used.
+menu:
+  intro:
+    parent: concepts
+    weight: 10
+
+aliases:
+  - /updateplans/
+---
+
+{{% notes type="warning" %}}
+Update plans are currently in experimental preview and will only show up in `--help` if the environment variable `PULUMI_EXPERIMENTAL` is set to `true`.
+{{% /notes %}}
+
+The result of previews can be saved to an _update plan_, this can then be used during a later update to constrain the update only to the operations that were planned.
+
+## Generate an update plan
+
+To generate an update plan run `pulumi preview --save-plan=plan.json`. The plan will be written to the `plan.json` file.
+
+## Use an update plan
+
+To use an update plan run `pulumi up --plan=plan.json`, this will constrain the update to only the operations that were saved to the plan.
+
+## Format
+
+The format of update plans is currently experimental and subject to change. The most up to date description of the format is the [code](https://github.com/pulumi/pulumi/blob/master/sdk/go/common/apitype/plan.go).
```

Comment:
If this includes deprecation in full, we should explicitly state this as well just so folks know how much they can reasonably expect to rely on the experimental feature. If we have an official policy on experimental feature non/support, it would be good to link here. If we don't have one, would be good to write one (when we have time).

Reply:
> If we have an official policy on experimental feature non/support, it would be good to link here.

Yeh that would be good! Raised https://github.com/pulumi/pulumi-hugo/issues/2239